lib/Parse/ParseDecl.cpp

```diff
@@ -3703,6 +3697,12 @@ ParserStatus Parser::parseLineDirective(bool isLine) {
                                              Filename.getValue(), LineOffset);
   assert(isNewFile);(void)isNewFile;
 
+  // Lex of next token must be deferred until after virtual file set up.
```

Comment:
Nit: lex is a verb.
Nit: setup is one word when used as a noun.

Reply:
Fixed in an amend commit